### PR TITLE
Development

### PR DESCRIPTION
**Pull request by radumojic**
### Staking V4 Redesign: Step 3

- Snow an overview of the grouped nodes on Identity Details | Provider Details and Account/Nodes pages
- Group the nodes by status
- Show a tooltip containing all the necessary information

### General

- Mobile wrapping fixes
- Updated NodeInformation display ( show Auction status )
- Add an extra check on auction list display to avoid display issues due to malformed api responses from cache
- Renamed NodeThreshold into AuctionThreshold to better reflect the specific functionality in that component
- Added a new component, NodeThreshold to handle pre-processing required on a node in order to properly display the dropped nodes
- Added the useIsAuctionNodeDropped hook - don't show the qualified topup on dropped nodes as is was redistributed among other nodes

### CODE REVIEW

File: src/components/Nodes/NodeLockedStake/NodeLockedStake.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import BigNumber from 'bignumber.js';
+
+import { Overlay, FormatAmount, NodeLockedStakeTooltip } from 'components';
+import { useIsAuctionNodeDropped } from 'hooks';
+import { NodeStatusEnum, NodeType } from 'types';
+
+export interface StandardRowUIType {
```
Comment (arhtudormorar) [inline src/components/Nodes/NodeLockedStake/NodeLockedStake.tsx:7]: only export interfaces if used outside

File: src/components/Nodes/NodesStatusPreviewCards/components/NodePanel/NodePanelCards.tsx [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+  NodeRating,
+  NodeStatus,
+  ShardLink
+} from 'components';
+import { formatBigNumber } from 'helpers';
+import { useGetEpochRemainingTime, useIsSovereign } from 'hooks';
+import { nodesOverviewSelector } from 'redux/selectors';
+import { NodeType, NodeStatusEnum, WithClassnameType } from 'types';
+
+export interface PanelCardUIType extends WithClassnameType {
```
Comment (arhtudormorar) [inline src/components/Nodes/NodesStatusPreviewCards/components/NodePanel/NodePanelCards.tsx:17]: best to keep one component / file

File: src/components/Nodes/NodeLockedStake/NodeLockedStake.tsx (hunk 1 above)
Comment (radumojic) [reply src/components/Nodes/NodeLockedStake/NodeLockedStake.tsx]: updated

**Review by ioandanpopa — APPROVED** [review]

File: src/components/Nodes/NodesStatusPreviewCards/components/NodePanel/NodePanelCards.tsx (hunk 2 above)
Comment (radumojic) [reply src/components/Nodes/NodesStatusPreviewCards/components/NodePanel/NodePanelCards.tsx]: updated

**Review by CiprianDraghici — APPROVED** [review]

**Pull request merged**